frontend/src/libs/components/router/router.tsx

```diff
@@ -49,6 +50,7 @@ const Router = (): JSX.Element => {
         <Route path={AppRoute.SIGN_UP_BUSINESS} element={<Auth />} />
         <Route path={AppRoute.SIGN_UP_CUSTOMER} element={<Auth />} />
         <Route path={AppRoute.ORDER} element={<Order />} />
+        <Route path={AppRoute.DRIVER_ORDER} element={<DriverOrder />} />
```

Comment:
Now this page corresponds to newly merged `  <Route
            path={AppRoute.ORDERS}
            element={
              <PageLayout isSidebarHidden>
                <Orders />
              </PageLayout>
            }
          />`.
So, the whole JSX of DriverOrder should be moved into `<Orders/>`. But I agree that it will be better if we use `AppRoute.DRIVER_ORDER` as path name instead of `AppRoute.ORDERS`, and component name as `DriverOrder` instead of `Orders`.